src/App.tsx

```diff
@@ -76,27 +75,17 @@ const App: React.FC = () => {
 
   const unconfigureServer = useCallback(async () => {
     try {
-      const response = await fetchJSON<UnconfigureResponse>(
-        '/scan/unconfigure',
-        {
-          method: 'post',
-        }
-      )
-      if (response.status === 'ok') {
-        unconfigure()
-      }
+      await deleteElection()
+      setElection(undefined)
     } catch (error) {
       console.log('failed unconfigureServer()', error) // eslint-disable-line no-console
     }
-  }, [unconfigure])
+  }, [setElection])
 
-  const fetchElection = useCallback(
-    async () =>
-      fetchJSON<CardReadLongResponse>('/card/read_long').then((card) =>
-        JSON.parse(card.longValue)
-      ),
-    []
-  )
+  const fetchElection = useCallback(async () => {
```

Comment:
should we use the opportunity to clarify this function as `loadElectionFromCard` or the like?

Reply:
Done